### PR TITLE
- Create SipCalculator screen

### PR DESCRIPTION
**Pull request by Parthshiroya06**

### CODE REVIEW

**Comment by bharathkumarchowdary** [comment]
On the iOS app, I face the following error

Unable to resolve module react-native-chart-kit from /Users/bharathkumargundala/code/lekkalu-app/app/(drawer)/SipCalculator/index.tsx: react-native-chart-kit could not be found within the project or in these directories:
  node_modules
  
  3 | import { Controller, useForm } from 'react-hook-form'
  4 | import { Dimensions, useWindowDimensions } from 'react-native'
> 5 | import { PieChart } from 'react-native-chart-kit'
    |                           ^
  6 | export default function Page() {
  7 |   const { width } = useWindowDimensions()
  8 |   const { handleSubmit, register, control, setValue, getValues, watch } = useForm({

ABI49_0_0RCTFatal
ABI49_0_0RCTConvertArrayValue
89420547-98A3-352A-AD2B-BBCB6006FB29
89420547-98A3-352A-AD2B-BBCB6006FB29
89420547-98A3-352A-AD2B-BBCB6006FB29
_dispatch_main_queue_callback_4CF
B37A1273-B0C2-312E-9E0B-7143FB645C69
B37A1273-B0C2-312E-9E0B-7143FB645C69
CFRunLoopRunSpecific
GSEventRunModal
F1A8BD61-9CF2-3054-B622-CB63D6CEB4CB
UIApplicationMain
main
A795AD37-6266-385E-95E6-C1CA804C926E

**Comment by rockingrohit9639** [comment]
> On the iOS app, I face the following error
> 
> Unable to resolve module react-native-chart-kit from /Users/bharathkumargundala/code/lekkalu-app/app/(drawer)/SipCalculator/index.tsx: react-native-chart-kit could not be found within the project or in these directories: node_modules
> 
> 3 | import { Controller, useForm } from 'react-hook-form' 4 | import { Dimensions, useWindowDimensions } from 'react-native'
> 
> > 5 | import { PieChart } from 'react-native-chart-kit'
> > |                           ^
> > 6 | export default function Page() {
> > 7 |   const { width } = useWindowDimensions()
> > 8 |   const { handleSubmit, register, control, setValue, getValues, watch } = useForm({
> 
> ABI49_0_0RCTFatal ABI49_0_0RCTConvertArrayValue 89420547-98A3-352A-AD2B-BBCB6006FB29 89420547-98A3-352A-AD2B-BBCB6006FB29 89420547-98A3-352A-AD2B-BBCB6006FB29 _dispatch_main_queue_callback_4CF B37A1273-B0C2-312E-9E0B-7143FB645C69 B37A1273-B0C2-312E-9E0B-7143FB645C69 CFRunLoopRunSpecific GSEventRunModal F1A8BD61-9CF2-3054-B622-CB63D6CEB4CB UIApplicationMain main A795AD37-6266-385E-95E6-C1CA804C926E

Have you done `npm install` before running the project ?

**Comment by bharathkumarchowdary** [comment]
yes @rockingrohit9639 , I did nom install

**Comment by mohitkhatri06** [comment]
in android it feels very heavy,  not too smooth draggable range.

File: utils/theme.ts [hunk 1]
```diff
@@ -14,4 +14,9 @@ export const THEME_COLORS = {
     800: '#113053',
     900: '#102945',
   },
+  green:{
+    50:"#79D40A",
+    100:"#A8EE54",
+    200:"#91F716"
```
Comment (rockingrohit9639) [inline utils/theme.ts:20]: `native-base` already have these colors... we don't have to explicitely define them

File: app/login.tsx [hunk 2]
```diff
@@ -21,11 +21,11 @@ export default function Login() {
...
   const handleLogin = (values: LoginSchema) => {
     loginMutation.mutate(values)
-    router.replace('/(drawer)/home')
+    router.replace('/(drawer)/SipCalculator')
   }
 
-  if (tokenData) {
-    return <Redirect href="/dashboard" />
+  if (!tokenData) {
+    return <Redirect href="/(drawer)/SipCalculator/" />
```
Comment (rockingrohit9639) [inline app/login.tsx:28]: No need to change this

**Review by rockingrohit9639 — COMMENTED** [review]
files names need to be changed.. there will be no files like `(drawer)/some-file`
instead create files like
`calculators/sip.tsx`

File: utils/theme.ts (hunk 1 above)
Comment (Parthshiroya06) [reply utils/theme.ts:20]: ok, i am remove it.

File: app/login.tsx (hunk 2 above)
Comment (Parthshiroya06) [reply app/login.tsx]: i have change dashboard as first screen. please verify.

**Pull request closed without merging**